libs/sdk-backend-bear/src/convertors/dateFormatting/dateValueParser.ts

```diff
@@ -0,0 +1,22 @@
+// (C) 2020 GoodData Corporation
+import parse from "date-fns/parse";
+import { UnexpectedError } from "@gooddata/sdk-backend-spi";
+
+export const dateFormats = ["MM/dd/yyyy", "dd/MM/yyyy", "dd-MM-yyyy", "yyyy-MM-dd", "M/d/yy", "dd.MM.yyyy"];
```

Comment:
You need to add the `as const` at the end. Without it, the DateFormat type will be just `string`, not only the allowed values.

Reply:
Thanks, I've updated the PR, please help me review again.